torch/csrc/jit/mobile/import.h

```diff
@@ -85,6 +89,8 @@ c10::intrusive_ptr<c10::ivalue::Object> objLoaderMobile(
     at::StrongTypePtr type,
     at::IValue input,
     std::shared_ptr<mobile::CompilationUnit> mobile_compilation_unit);
+bool tensorInConstantsArchive(
```

Comment:
maybe IsTensorIn...

some docs would be nice too.

Reply:
Rename to `isTensorInBytecodeArchive` in latest version. As a note, it is *not* `isTensorInConstantArchive`. I realize that there are always tensors in `constants` archive, but not `bytecode` archive in the future model, so I change the logic.

Reply:
In terms of why unit test didn't catch it? Likely all tensors in existing testing models from `bytecode` are exactly the same as `constants`...anyway, it's better to use `isTensorInBytecodeArchive` logic.